Jenkinsfile

```diff
@@ -33,29 +33,31 @@ def vagrantfile = '''
 require 'uri'
 
 node_id = URI::encode(ENV['NODE_ID'] || '')
-name = "mojave-xcode11-#{ENV['BRANCH_NAME'] || ''}"
+name = "catalina-xcode11.3-#{ENV['BRANCH_NAME'] || ''}"
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "mojave-xcode11"
+    config.vm.box = "catalina-xcode11.3"
+    config.vm.synced_folder ".", "/vagrant", disabled: true
```

Comment:
Because Catalina makes the system volume read only. when vagrant tries to sync, it will fail because the system volume is read only. It is enabled by default, so I have to disable it.